pkg/webui/console/containers/join-eui-prefixes-input/join-eui-prefixes-input.js

```diff
@@ -179,6 +186,19 @@ class JoinEUIPrefixesInput extends React.PureComponent {
           onBlur={this.handleBlur}
           error={error}
         />
+        <button
+          title={m.zeroInput}
+          type="button"
+          className={style.buttonZeros}
+          onClick={this.handleZerosClick}
+        >
+          <Icon
+            className={style.buttonIcon}
+            onClick={this.handleZerosClick}
+            small
+            icon="exposure_zero"
+          />
```

Comment:
Use a custom `<span>00</span>` instead of the icon and style it to use `$font-family-mono`.